app/views/templates/_history.html.erb

```diff
@@ -0,0 +1,25 @@
+<% unless history.empty? %>
+  <% history.each do |audit| %>
```

Comment:
isnt the above line redundant? if its an empty array then each wont do anything.